dask_kubernetes/core.py

```diff
@@ -399,8 +399,7 @@ def _delete_pods(self, to_delete):
             try:
                 self.core_api.delete_namespaced_pod(
                     pod.metadata.name,
-                    self.namespace,
-                    kubernetes.client.V1DeleteOptions()
+                    self.namespace
```

Comment:
Should we support kubernetes < 9 as well with a version check? I'm not sure how big of a deal requiring kubernetes >= 9.0 is. cc @mrocklin.

Reply:
The library should be [backward compatible](https://github.com/kubernetes-client/python#compatibility-matrix). So I don't see it being a big deal.